components/org.wso2.carbon.identity.auth.service/src/main/java/org/wso2/carbon/identity/auth/service/util/Constants.java

```diff
@@ -58,6 +58,7 @@ public class Constants {
     public final static String RESOURCE_ORGANIZATION_ID = "resourceOrgId";
     public static final String AUTHENTICATION_TYPE = "authenticationType";
     public final static String VALIDATE_LEGACY_PERMISSIONS = "validateLegacyPermissions";
+    public final static String TENANT_PERSPECTIVE_REQUEST_REGEX = "^/t/[^/]+/o/[a-f0-9\\-]+?";
```

Comment:
Let's have the Pattern as the constant as the regex is constant, so it will be pre compiled in class well.

Reply:
Done